app/recaptcha/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Google Inc.
```

Comment:
Please don't add headers to the files under app/recaptcha. Those files are kinda a weird situation (as described in app/recaptcha/README.md).

Reply:
Acknowledged